endpoint.go

```diff
@@ -462,49 +342,59 @@ func (ep *endpoint) hasInterface(iName string) bool {
 	return false
 }
 
-func (ep *endpoint) Leave(containerID string, options ...EndpointOption) error {
-	var err error
-
+func (ep *endpoint) Leave(sbox Sandbox, options ...EndpointOption) error {
 	ep.joinLeaveStart()
 	defer ep.joinLeaveEnd()
 
-	ep.processOptions(options...)
+	if sbox == nil || sbox.Info().ID() == "" || sbox.Info().Key() == "" {
+		return types.BadRequestErrorf("invalid container passed to enpoint leave: %v", sbox)
+	}
+
```

Comment:
That's true the sandbox presence check is missing. 
Will take care of this.
Thanks